ui/tests/acceptance/vault-reporting/index-test.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { visit, currentURL, waitFor } from '@ember/test-helpers';
+import { login } from 'vault/tests/helpers/auth/auth-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+const mockedEmptyResponse = {
+  data: {
+    auth_methods: {},
+    kvv1_secrets: 0,
+    kvv2_secrets: 0,
+    lease_count_quotas: {},
+    leases_by_auth_method: {},
+    replication_status: {},
+    secret_engines: {},
+  },
+};
+
+const mockedResponseWithData = {
+  data: {
+    auth_methods: { cats: 42 },
```

Comment:
Are the keys here auth method types or paths? I'm guessing paths because of the numerical number. For posterity it might be helpful to name these more similar to the real-world.